thirdParty/cmake-modules/FindADIOS.cmake

```diff
@@ -28,42 +29,92 @@
 # This module will define the following variables:
 #   ADIOS_INCLUDE_DIRS    - Include directories for the ADIOS headers.
 #   ADIOS_LIBRARIES       - ADIOS libraries.
+#   ADIOS_DEFINITIONS     - ADIOS compile definitions.
 #   ADIOS_FOUND           - TRUE if FindADIOS found a working install
 #   ADIOS_VERSION         - Version in format Major.Minor.Patch
+#   ADIOS_HAVE_SEQUENTIAL - TRUE if found library links as sequential only
 #
 # Not used for now:
 #   ADIOS_DEFINITIONS     - Compiler definitions you should add with
 #                           add_definitions(${ADIOS_DEFINITIONS})
 #
+# Example to find ADIOS (default)
+# find_package(ADIOS)
+# if(ADIOS_FOUND)
+#   include_directories(${ADIOS_INCLUDE_DIRS})
+#   add_executable(foo foo.c)
+#   target_link_libraries(foo ${ADIOS_LIBRARIES})
+# endif()
+
+# Example to find ADIOS using component
+# find_package(ADIOS COMPONENTS fortran)
+# if(ADIOS_FOUND)
+#   include_directories(${ADIOS_INCLUDE_DIRS})
+#   add_executable(foo foo.c)
+#   target_link_libraries(foo ${ADIOS_LIBRARIES})
+# endif()
+###############################################################################
+#Copyright (c) 2014, Axel Huebl and Felix Schmitt from http://picongpu.hzdr.de
```

Comment:
I am confused about the years. It seemingly went from `2014-2015` to just `2014`